apps/web/src/config/constants/endpoints.ts

```diff
@@ -14,6 +14,7 @@ export const GRAPH_API_POTTERY = 'https://api.thegraph.com/subgraphs/name/pancak
 export const ONRAMP_API_BASE_URL = 'https://pcs-on-ramp-api.com'
 export const TRANSAK_API_BASE_URL = 'https://api-stg.transak.com/api/v1'
 export const MOONPAY_BASE_URL = 'https://api.moonpay.com'
+export const NOTIFICATION_HUB_BASE_URL = 'https://lobster-app-6lfpi.ondigitalocean.app'
```

Comment:
what's the production domain?

Reply:
will push shortly. currently deploying with cannoli